src/nebula-hooks/use-virtualized-table.ts

```diff
@@ -53,7 +53,7 @@ const useVirtualizedTable = ({
   viewService,
 }: UseVirtualizedTable) => {
   const isPrintingMode = isPrinting(layout, viewService);
-  const shouldRender = layout.usePagination === false && !isPrintingMode;
+  const shouldRender = layout.usePagination === false || isPrintingMode;
```

Comment:
Should use only layout.usePagination to test:
const shouldRender = layout.usePagination === false;

Reply:
@quanho do you mean remove `isPrintingMode;` check completely?

Reply:
Yes, I used isPrintingMode to do the trick of use pagination table to render virtualized table snapshot.
Now if we always use virtualized table to render virtualized table snapshot and pagination table to render pagination table snapshot then we don't need to check if it is in printing mode or not.
You should remove checking isPrintingMode everywhere.